index.html

```diff
@@ -318,7 +318,7 @@ <h2>Contact me</h2>
                                 required>
                             <textarea type="message" name="message" class="message"
                                 placeholder="Hey I'd like to connect to..." maxlength="500" required></textarea>
```

Comment:
Great job on your contact section! I have a few suggestions **to help you follow `HTML` best practices and make it even better** to match the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122&t=S6boPW3BisotT0aZ-0):

- [ ] Could you kindly center the form? I noticed that it's a bit to the left, and this might be due to the `margin-left` property you added to your section with **the contact class**. 🙌

- [ ] I suggest removing the duplicated "type" property in your input for `name`. You currently have `type='name'` and `type='text'` which **isn't a valid input type**. Please remove `type='name'` and keep `type='text'`. 💯

- [ ] Also, kindly check the width and padding of your "Get in touch" button to ensure that it matches the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122&t=S6boPW3BisotT0aZ-0). 🔍

- [ ] Additionally, it's not necessary to include the `type` property in your `textarea`. You can learn more about this in the following [link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea). 🎯

  ### Your design:
  ![image](https://github.com/pradmon/My-Portfolio/assets/107952724/e45e8a36-9955-4515-b182-1122cfdb9365)

  ### Figma design:
  ![image](https://github.com/pradmon/My-Portfolio/assets/107952724/094c7900-1297-4cac-85f0-16e93e1a66b5)